### PR TITLE
xlsxfilename, roundfloat, format

### PR DESCRIPTION
**Pull request by FV1932**
change sleep time to 1sec
change local xlsx file name
change round-while-store to format-when-print

### CODE REVIEW

File: b616.py [hunk 1]
```diff
@@ -82,16 +82,13 @@ def get_desc_list():
                 "Your score of",
                 error_row[0],
                 "is",
-                int(error_row[3]),
+                error_row[3],
                 ", which might be invaild, please check xlsx then run again",
```
Comment (JacobLiu001) [inline b616.py]: spell mastar

File: b616.py [hunk 2]
```diff
@@ -126,46 +123,32 @@ def write_ptt_history_csv():
...
+    rows = desc_ra_list[:custom_num]
+    for row in rows[:30]:
+        print_row(row)
     print()
-    if row_num == 30:
+
+    if custom_num_over30:
         print("b30底分:", f"{b30_only:.4f}", " (忽略r10)")
-        print("不推b30, 也就是r10=b10时的理论最高ptt: ", f"{b30_withr10:.4f}")
+        print("r10=b10时的理论最高ptt: ", f"{b30_withr10:.4f}")
```
Comment (JacobLiu001) [inline b616.py]: Why two things here

File: b616.py [hunk 3]
```diff
@@ -126,46 +123,32 @@ def write_ptt_history_csv():
...
-        print()
-        for row_num in range(30, custom_num):
-            print_rows(desc_ra_list)
         print()
+
+    for row in rows[30:]:
+        print_row(row)
+
+    if custom_num != 30:
         print(f"b{custom_num}底分:", f"{cust_average:.4f}", " (忽略r10)")
```
Comment (JacobLiu001) [inline b616.py]: again, this can be just one f-string

File: b616.py (hunk 1 above)
Comment (FV1932) [reply b616.py]: wait what

Comment (JacobLiu001) [reply b616.py]: "invaild"

Comment (FV1932) [reply b616.py]: oh invaild

File: b616.py [hunk 4]
```diff
@@ -334,19 +343,21 @@ def draw_history_b30_chart():
...
-    dot_size = max((50 - pow(len(y_maxiptt), 0.5)), 10)
+    y_baseptt = line[3]
     plt.scatter(x_time, y_realptt, s=dot_size)
+    plt.scatter(x_time, y_baseptt, s=dot_size)
     plt.scatter(x_time, y_maxiptt, s=dot_size)
     plt.tick_params(axis="x", labelrotation=61.6)
     plt.xlabel("年/月/日", fontsize=12)
     plt.ylabel("ptt", fontsize=12)
-    plt.legend(["实际输入ptt", "理论最高ptt"], loc="best")
+    plt.legend(["真实ptt", "仅底分ptt", "理论最高ptt"], loc="best")
```
Comment (JacobLiu001) [inline b616.py]: 这么写也行, 但是我个人习惯直接在plt.scatter传入label

Comment (FV1932) [reply b616.py]: works, 这个legend按照位置对应过去确实不是好方案, 我下次修改

**Comment by JacobLiu001** [comment]
@FV1932 If you made any changes remember to push your changes

**Pull request merged**